e2e/jest-e2e.json

```diff
@@ -0,0 +1,14 @@
+{
+  "moduleFileExtensions": [
+      "ts",
```

Comment:
we already have the `jest.config.js` file. I'm not sure this file is needed though

Reply:
yes, this file and e2e folder were committed by mistake. Removed